woocommerce/woocommerce-android
# TTP: Adds a TPP button to the payments hub

Closes: #8368 


### Description

Adds TPP button to the payment hub. The click on the button does nothing so far

### Testing instructions

* If TPP is available on the device - notice that there is a new TPP row on the payments hub
* If TPP is not available on the device - notice that the new row is not visible. Make sure that it is not visible on the release build

### Images/gif

<img src="https://user-images.githubusercontent.com/4923871/220566306-59595a99-6fbb-408f-b982-91889d96c719.png" width=400/>
<img src ="https://user-images.githubusercontent.com/4923871/220566325-7b1de351-6923-4f0e-bbcf-16d639333d76.png" width=400/>
